build.gradle

```diff
@@ -998,6 +973,9 @@ project(':core') {
   if (userEnableTestCoverage) {
     scoverage {
       scoverageVersion = versions.scoverage
+      if (versions.baseScala == '2.13') {
+        scoverageScalaVersion = '2.13.8' // there's no 2.13.10 artifact, org.scoverage:scalac-scoverage-plugin_2.13.8:1.4.11 is the latest as of now
```

Comment:
Does this mean that the coverage will use 2.13.8 while compilation still uses 2.13.10 or whatever is set by gradle?

Reply:
```
scoverage - Scoverage dependencies
+--- org.scoverage:scalac-scoverage-plugin_2.13.8:1.4.11
|    +--- org.scala-lang:scala-library:2.13.8 -> 2.13.12
|    \--- org.scala-lang.modules:scala-xml_2.13:1.3.0
|         \--- org.scala-lang:scala-library:2.13.1 -> 2.13.12
\--- org.scoverage:scalac-scoverage-runtime_2.13:1.4.11
     \--- org.scala-lang:scala-library:2.13.8 -> 2.13.12
```